README.md

```diff
@@ -48,17 +48,17 @@ SETUP
     
      - When debug option is enabled(```DEBUG=True```), DO NOT forget to add urlpatterns as shown below:
      
-            from django.conf import settings
-            from django.conf.urls.static import static
+           from django.conf import settings
+           from django.conf.urls.static import static
             
-            if settings.DEBUG:
+           if settings.DEBUG:
                 urlpatterns += static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
```

Comment:
if you would like to pull the above lines, then I think `urlpatterns += …` also should be pulled to left.

Reply:
I corrected as you said. Is it possible to check?